.github/workflows/build.yml

```diff
@@ -10,90 +10,36 @@ on:
   push:
     paths-ignore:
       - '*.md'
-    branches:
-      - master
 jobs:
 
   # Build and test plugin and provide artifact.
   build:
     name: Build
-    runs-on: ubuntu-latest
+    runs-on: codebuild-ion-intellij-plugin-${{ github.run_id }}-${{ github.run_attempt }}-arm-3.0-large
```

Comment:
Using CodeBuild here is awesome. This makes me think it might be a good idea to use CodeBuild runner in [Ion-Java-Regression-Detection](https://github.com/amazon-ion/ion-java/blob/master/.github/workflows/ion-java-performance-regression-detector.yml) workflow too.

Reply:
Possibly. We can talk about it next week if you like.